core/server.go

```diff
@@ -747,11 +759,13 @@ func (s *Server) processSignatoryMessage(msg []byte) {
 
 	if len(resp.Error) > 0 {
 		log.Info("signatory response error", "err", resp.Error)
+		signingRequest.registerBundleError(signingRequest.userOpHash, ErrSigningFailure.AddMessage(resp.Error))
```

Comment:
Since we have already closed doneChan at this point, we cannot give a chance to other signatories which could be connected.

Reply:
good catch, changed in https://github.com/FastLane-Labs/atlas-operations-relay/pull/60/commits/7dd8c1edd8d65cf882ebad4b3114713fcdd426af, made the change for bundlers messages as well